mods/ra/rules/campaign-rules.yaml

```diff
@@ -67,6 +67,16 @@ FCOM:
 	-GivesBuildableArea:
 	BaseProvider:
 		Range: 0
+	RepairableBuilding:
+		RepairStep: 700
+		PlayerExperience: 25
+		RepairingNotification: Repairing
+	WithBuildingRepairDecoration:
+		Image: allyrepair
+		Sequence: repair
+		Position: Center
+		Palette: player
+		IsPlayerPalette: True
```

Comment:
Does this really belong in campaign-rules.yaml?

Reply:
Added this in response to a comment from @cjshmyr:

> In the originals the command center is repairable, but in here (and Soviet 02) it is not. Personally it felt odd I could not repair a base building, but I won't argue either way.

If we add this here at all we'll want it repairable in all missions.